.github/workflows/merge-pr.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         uses: actions/checkout@v3
       - name: Output Metadata
         # Run the cargo command and ignore any extra lines outside of the json result
-        run: CARGO_INCREMENTAL=0 RUSTFLAGS="-D warnings" cargo run --features frequency export-metadata ./js/api-augment/metadata.json
+        run: CARGO_INCREMENTAL=0 RUSTFLAGS="-D warnings" cargo run --features frequency-rococo-local -- export-metadata  --chain=frequency-local --tmp ./js/api-augment/metadata.json
```

Comment:
Switching the metadata export over to frequency-rococo-local. This is because it is still the all inclusive version of rococo and mainnet.